images/linux/scripts/installers/git.sh

```diff
@@ -46,7 +46,15 @@ DocumentInstalledItem "Git Large File Storage (LFS) ($(git-lfs --version 2>&1 |
 DocumentInstalledItem "Git-ftp ($(git-ftp --version | cut -d ' ' -f 3))"
 
 #Install hub
-snap install hub --classic
+TEMP_HUB=$(mktemp -d -t)
+pushd "$TEMP_HUB" || exit
```

Comment:
I suggest to remove `|| exit` here and in `popd` command and just run `set -e` at the begin of script

Reply:
done